app/models/ipeds_ic_ay.rb

```diff
@@ -2,6 +2,8 @@
 class IpedsIcAy < ActiveRecord::Base
   include Loadable, Exportable
 
+  USE_COLUMNS = [:tuition_in_state, :tuition_out_of_state, :books].freeze
```

Comment:
I disagree...the only exact duplication I see is between the two `ipeds` models. Anyway this seems like something that benefits from code locality - being able to look at the model and see in the same file which columns are actually consumed into the institutions table.

I do think maybe the way these constants are used in institution_builder could be refactored to better encapsulate this repeated pattern:
`columns = Crosswalk::USE_COLUMNS.map(&:to_s).map { |col| %("#{col}" = crosswalks.#{col}) }.join(', ')`